packages/api/tests/models/test-rules-model.js

```diff
@@ -253,7 +253,7 @@ test.serial('Creating a kinesis rule where an event source mapping already exist
             {
               UUID: randomString(),
               EventSourceArn: item.rule.value,
-              FunctionArn: `arn:aws:lambda:us-west-2:123456789012:function:${process.env.messageConsumer}`,
+              FunctionArn: `arn:aws:lambda:us-west-2:000000000000:function:${process.env.messageConsumer}`,
```

Comment:
I know the other was fake, but it was caught by git-secrets, and this is explicitly allowed.